pkg/testing/integration/program.go

```diff
@@ -1673,7 +1672,7 @@ func (pt *programTester) prepareDotNetProject(projinfo *engine.Projinfo) error {
 			return errors.Wrap(err, "failed to find a local Pulumi NuGet package")
 		}
 		if len(matches) != 1 {
-			return errors.New(fmt.Sprintf("attempting to find a local Pulumi NuGet package yielded %v results", matches))
+			return errors.New(fmt.Sprintf("attempting to find a local Pulumi NuGet package yielded %d results", len(matches)))
```

Comment:
Was this change needed?  It’s useful to know what the matches are when you hit this.

Reply:
It wasn't - this was part of debugging - removed now